data/org.freedesktop.impl.portal.FileChooser.xml

```diff
@@ -211,5 +211,95 @@
       <annotation name="org.qtproject.QtDBus.QtTypeName.Out1" value="QVariantMap"/>
       <arg type="a{sv}" name="results" direction="out"/>
     </method>
+    <!--
+      SaveFiles:
+      @parent_window: Identifier for the application window, see <link linkend="parent_window">Common Conventions</link>
+      @title: Title for the file chooser dialog
+      @options: Vardict with optional further information
+      @handle: Object path for the #org.freedesktop.portal.Request object representing this call
+
+      Asks for a folder to save a list of files.
+
+      Supported keys in the @options vardict include:
+      <variablelist>
+        <varlistentry>
+          <term>handle_token s</term>
+          <listitem><para>
+            A string that will be used as the last element of the
+            @handle. Must be a valid object path element. See the
+            #org.freedesktop.portal.Request documentation for more
+            information about the @handle.
+          </para></listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>accept_label s</term>
+          <listitem><para>
+            Label for the accept button. Mnemonic underlines are allowed.
+          </para></listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>modal b</term>
+          <listitem><para>
+            Whether the dialog should be modal. Default is yes.
+          </para></listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>choices a(ssa(ss)s)</term>
+          <listitem><para>
+            List of serialized combo boxes.
+            See org.freedesktop.portal.FileChooser.OpenFile() for details.
+          </para></listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>current_folder ay</term>
+          <listitem>
+            <para>
+              Suggested folder to save the files in. The byte array is
+              expected to be null-terminated.
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+        <term>files aay</term>
+          <listitem>
+            <para>
+              An array of file names to be saved. The array and byte
+              arrays are expected to be null-terminated.
+            </para>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+
+      The following results get returned via the
+      #org.freedesktop.portal.Request::Response signal:
+      <variablelist>
+        <varlistentry>
+          <term>uris as</term>
+          <listitem><para>
+            An array of strings containing the uri of each given
+            file. The path name of each will have a name matching the
+            given list of files.
```

Comment:
Should probably spell out here that the returned uris are in 1-1 correspondence to the passed-in files

Reply:
I've just pushed update that include improved doc comments. What do you think?